executor/common_linux.h

```diff
@@ -107,7 +107,7 @@ static bool write_file(const char* file, const char* what, ...)
 }
 #endif
 
-#if SYZ_EXECUTOR || SYZ_NET_DEVICES || SYZ_NET_INJECTION || SYZ_DEVLINK_PCI || __NR_syz_80211_join_ibss || __NR_syz_80211_inject_frame
+#if SYZ_EXECUTOR || SYZ_NET_DEVICES || SYZ_NET_INJECTION || SYZ_DEVLINK_PCI || SYZ_WIFI || __NR_syz_80211_inject_frame || __NR_syz_80211_join_ibss
```

Comment:
Please squash this commit with the one that adds the pseudo-syscalls, otherwise it's hard to review.

Reply:
Or put the config option commit first with empty implementation of pseudo-syscalls, and then fill in the implementation in another commit.